scripts/build/TestImpactAnalysis/tiaf.py

```diff
@@ -161,9 +161,9 @@ def _generate_result(self, s3_bucket: str, suite: str, return_code: int, report:
         result["change_list"] = self._change_list
         return result
 
-    def run(self, commit: str, src_branch: str, dst_branch: str, s3_bucket: str, s3_top_level_dir: str, suite: str, test_failure_policy: str, safe_mode: bool, test_timeout: int, global_timeout: int):
+    def run(self, commit: str, src_branch: str, dst_branch: str, s3_bucket: str, s3_top_level_dir: str, suite: str, test_failure_policy: str, safe_mode: bool, test_timeout: int, global_timeout: int, exclude_file: str):
```

Comment:
I think that auto-formatting to PEP8 is a great idea in general. In this specific case I am also considering using kwargs instead of a large number of parameters as we move to a more library like structure and our tiaf program is starts to require more and more arguments.

Reply:
This change would be made in a more substantial pr that incorporates a larger refactor of the code however.

Reply:
>This change would be made in a more substantial pr that incorporates a larger refactor of the code however.

Reasonable conclusion, feel free to add a task for it to pick it up at a later date 👍